src/mailadm/bot.py

```diff
@@ -34,15 +34,28 @@ def __init__(self, db: DB, account: deltachat.Account):
         self.account = account
         with self.db.read_connection() as conn:
             config = conn.config
-            self.admingrpid = config.admingrpid
+            self.admingrpid = int(config.admingrpid)
 
     @account_hookimpl
     def ac_incoming_message(self, message: deltachat.Message):
         arguments = message.text.split(" ")
-        print("process_incoming command:", arguments)
+        print("process_incoming message:", message.text)
         if not self.check_privileges(message):
-            message.create_chat()
-            message.chat.send_text("Sorry, I only take commands from the admin group.")
+            if message.text[0] == "/":
+                message.create_chat()
```

Comment:
btw, iisic this would allow a sender to create a group as we don't restrict here to 1:1 chats.

Reply:
Ah, well that would become a spam nightmare in the admin group. I added a conditional which prevents this.

Reply:
A test for this would be nice. #todo